TypeCobol.LanguageServer.Test/LSRTests/ProcedureCompletionInputInoutOutput-NoContext/input/ProcedureCompletionInputInoutOutput-NoContext.tlsp

```diff
@@ -131,7 +131,7 @@
     },
     {
       "category": 2,
-      "message": "{\"jsonrpc\":\"2.0\",\"id\":\"2\",\"result\":[{\"label\":\"BirthDate (DATE) (Person::BirthDate)\",\"kind\":6,\"insertText\":\"Person::BirthDate IN-OUT \"},{\"label\":\"MyDate (DATE) (MyDate)\",\"kind\":6,\"insertText\":\"MyDate IN-OUT \"}]}"
+      "message": "{\"jsonrpc\":\"2.0\",\"id\":\"2\",\"result\":[{\"label\":\"BirthDate (DATE) (Person::BirthDate)\",\"kind\":6,\"insertText\":\"Person::BirthDate\"},{\"label\":\"MyDate (DATE) (MyDate)\",\"kind\":6,\"insertText\":\"MyDate\"}]}"
```

Comment:
In this file, the completion results have changed. This is normal to have changes in error messages (textDocument/publishDiagnostics) but a bit strange to have changes in completion results. The grammar changed so I guess that the produced CodeElement/Nodes are now different and this affects the completion process. We should check whetehr the results are ok or not.

Reply:
Check on branch v1.4.1 displays the same diffene